src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/PackageLibs.targets

```diff
@@ -140,6 +140,12 @@
       <FilesToPackage Condition="'%(FilesToPackage.SubFolder)' != ''">
         <TargetPath>%(TargetPath)%(FilesToPackage.SubFolder)</TargetPath>
       </FilesToPackage>
+
+      <!-- Include source files for symbol packages. -->
+      <FilesToPackage Include="@(Compile->'%(FullPath)')" Condition="$([System.String]::new('%(FullPath)').StartsWith('$(ProjectDir)'))">
```

Comment:
Hmm, for corefx and wcf this means the root of the repo (so it would include sources under `Tools` and `bin`), and I didn't think much about the TFS case. I suppose for that I can find a different property that changes internally to the root of the enlistment and use that as the root here.